### DISCUSSION TITLE
kernel bermasalah

### DISCUSSION THREAD

**Discussion by yulekristina** [teamalgoritma/community#215] [Q&A] [answered]
Dear Bapak/Ibu,

Mohon bantuan perihal kernel di python saat membuka pandas. Awalnya bisa pas "Run All Above" data di pandas terbaca, tapi setelah bbrp menit pandas spt tidak tbaca saat di "Run" (error terlampir). Solusinya bagaimana ya? apakah hrs membuat environment baru lagi, mengingat itu perlu waktu juga. Terima kasih

<img width="589" alt="image" src="https://user-images.githubusercontent.com/107384192/180030073-8eda57b4-eeb8-4ed2-a910-c2795bd444f0.png">

**Comment by idoali** [comment, marked as answer]
Selamat pagi bu

Ini terjadi karena python tidak mengenal apa itu `pd` bu. `pd` sendiri adalah singkatan yang biasa kita gunakan untuk menyebut `pandas`. Tetapi sebelum menggunakan singkatan, terlebih dulu kita perlu **memperkenalkannya** ke python. Ini yang biasanya kita lakukan melalui code : 

```{}
import pandas as pd
```

Disini kita melakukan `import` library `pandas` yang kemudian kita perkenalkan sebagai `pd`. Setelah Ibu melakukan ini, barulah Ibu bisa menjalankan code yang Ibu tunjukkan di gambar. Semoga jawabannya bisa membantu.

**Comment by yulekristina** [comment]
Ijin btanya kak Ido, klo mo upload ulang capstone project di github harus
bikin repository baru lagi atau cukup pke repository yang lama ya?

**Comment by idoali** [comment]
Baik bu, terima kasih atas pertanyaannya. 

Jika ingin mengupload ulang, ibu bisa menguploadnya lagi di repository yang sama bu. Jika ada file yang namanya sama, maka file yang lama akan tergantikan dengan file yang baru. 

![tedsaw](https://user-images.githubusercontent.com/55035758/180207935-f2912988-88e5-4424-9332-20a7c3dcc8ba.png)

Semoga pertanyaannya bisa terjawab bu

**Comment by yulekristina** [comment]
Baik kak terima kasih, saya ada kendala lg pas copy ke visual studio
kak..kenapa ada message ini ya kak:

[image: image.png]
mohon arahannya mengingat hr ini batas waktu resubmit capstone.nya

**Reply by yulekristina** [reply]
Kalo kaya gini musti diapain ya kak, untuk membaca folder di visual studio code

<img width="848" alt="image" src="https://user-images.githubusercontent.com/107384192/180240201-0a6b6f3b-ab6a-4652-964f-8a3b1168325f.png">

**Reply by amaliasf** [reply]
Halo Bu @yulekristina , untuk membaca folder pada Visual Studio Code, Ibu bisa melakukan hal berikut:
1. Menuju ke menu `File > Add Folder to Workspace`
2. Pilih folder lalu `Add`

Semoga bisa membantu Bu :)

**Comment by yulekristina** [comment]
[image: image.png]
kalo kaya gini napa ya kak?
